CRM/Civicaseextras/APIWrappers/Case/Getcaselist.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class CRM_Civicaseextras_APIWrappers_Case_Getcaselist implements API_Wrapper {
+
+  /**
+   * {@inheritDoc} It Alters the Case Getcaselist action so it includes the case duration
+   * custom field to the response.
+   */
+  public function fromApiInput($apiRequest) {
+    if (!$this->shouldRun($apiRequest)) {
+      return $apiRequest;
+    }
+
+    $return = CRM_Utils_Array::value('return', $apiRequest['params'], []);
+    $caseDuration = _civicaseextras_get_caseDurationField();
```

Comment:
I see you changed this in `CRM_Civicaseextras_AngularModifiers_CaseDuration`, why not here too?